addons/stock/models/stock_inventory.py

```diff
@@ -311,7 +309,7 @@ def _get_exhausted_inventory_line(self, products, quant_products):
             vals.append({
                 'inventory_id': self.id,
                 'product_id': product.id,
-                'location_id': self.location_id.id,
+                'location_id': self.location_ids[0].id or None,
```

Comment:
i thought we removed the exhausted options?

Reply:
confirmed with lap, the exhausted code should be removed

Reply:
Same for all reference to inventory's `package_id`, `partner_id`, `lot_id` and `category_id` ?
Since it's not configurable in the inventory view and can't be used anymore, I presume I must delete those ones too.

Reply:
yes i think so